src/lua/loaders.lua

```diff
@@ -119,6 +91,33 @@ local function gen_search_func(path_fn, templates, need_traverse)
     end
 end
 
+-- Lua table with "package searchers".
+local searchers = debug.getregistry()._PACKAGE_SEARCHERS
+
+rawset(searchers, 'preload', function(name)
+    assert(type(package.preload) == 'table',
+           "'package.preload' must be a table")
+    local loader = package.preload[name]
+    if loader ~= nil then
+        return loader, ':preload:'
```

Comment:
I've seen several usages of the keyword `preload` (`:preload:`) or `builtin` (`:builtin:`) here. Maybe, it is better to use some constants for them (like `BUILTIN_SEARHER_NAME`)?
Feel free to ignore.

Reply:
This seems to be resolved in scope of **[PATCH 8/9] lua: implement common legacy loader generator** either, doesn't it?

Reply:
Yes, so feel free to ignore.